eclipse/src/saros/ui/wizards/pages/ConfigurationSummaryWizardPage.java

```diff
@@ -158,24 +159,25 @@ protected Composite createRightColumn(Composite composite) {
     skypeUsername = new SummaryItemComposite(networkSettingsComposite, SWT.NONE);
     skypeUsername.setLayoutData(new GridData(SWT.FILL, SWT.CENTER, true, false));
 
-    /*
-     * separator
-     */
-    new Label(rightColumn, SWT.SEPARATOR | SWT.HORIZONTAL)
-        .setLayoutData(new GridData(SWT.FILL, SWT.CENTER, true, false));
-
-    /*
-     * statistic settings
-     */
-    Composite statisticSettingsComposite = new Composite(rightColumn, SWT.NONE);
-    statisticSettingsComposite.setLayoutData(new GridData(SWT.FILL, SWT.CENTER, true, true));
-    statisticSettingsComposite.setLayout(LayoutUtils.createGridLayout(0, 5));
-
-    statisticSubmission = new SummaryItemComposite(statisticSettingsComposite, SWT.NONE);
-    statisticSubmission.setLayoutData(new GridData(SWT.FILL, SWT.CENTER, true, false));
-    errorLogSubmission = new SummaryItemComposite(statisticSettingsComposite, SWT.NONE);
-    errorLogSubmission.setLayoutData(new GridData(SWT.FILL, SWT.CENTER, true, false));
-
+    if (FeedbackManager.isFeedbackFeatureRequired()) {
```

Comment:
I guess it would have been easier just to hide the composite. But I do not know what the layout manager will do in such a case.